collection_requirements.rst

```diff
@@ -1,73 +1,76 @@
-*****************************
-Ansible Collections Checklist
-*****************************
+******************************************
+Ansible Community Collections Requirements
```

Comment:
To emphasize that this is not for Community Collections, but for collections included in the Ansible Community Package:
```suggestion
Ansible Community Package Collections Requirements
```

Reply:
Sounds good, committed, thanks!